2.3.3/bullseye/trivyignore

```diff
@@ -0,0 +1,5 @@
+# Airflow and FABs use of PyJWT isn't impacted
+CVE-2022-29217
```

Comment:
I don't think this should be needed anymore as (I think) we've upgraded FAB to 4.1.2 which has updated versions of this